src/_guides/language/language-tour.md

```diff
@@ -2149,11 +2149,10 @@ use the `rethrow` keyword.
 
 <?code-excerpt "misc/test/language_tour/exceptions_test.dart (rethrow)" replace="/rethrow;/[!$&!]/g"?>
 {% prettify dart %}
-final foo = '';
-
 void misbehave() {
   try {
-    foo = "You can't change a final variable's value.";
```

Comment:
Compile-time error prevents us from running this code. So I switch to code the would raise a runtime exception.